test/sales_engine_test.rb

```diff
@@ -4,7 +4,7 @@
 class SalesEngineTest < Minitest::Test
 
   def test_it_has_a_merchant_repository
-    data_location = {merchant: "./data/fixtures/merchants.csv", item: "./data/fixtures/items.csv"}
+    data_location = {merchant: "./data/fixtures/merchants.csv", item: "./data/fixtures/items.csv", customer: "./data/fixtures/customers.csv"}
```

Comment:
Line is too long. [141/80]<br>Space inside { missing.<br>Space inside } missing.